<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editwindow</class>
 <widget class="QDialog" name="editwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>景点名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>景点信息：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>添加新顶点</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_info">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_start">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>20</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_final">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>边名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_arc">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_length">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>长度：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加边</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>景点名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name_del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除顶点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>起点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_final_del">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_start_del">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>终点：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pathButton_del">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除边</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>editwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>editwindow</receiver>
   <slot>addnode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>editwindow</receiver>
   <slot>addpath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_del</sender>
   <signal>clicked()</signal>
   <receiver>editwindow</receiver>
   <slot>delnode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathButton_del</sender>
   <signal>clicked()</signal>
   <receiver>editwindow</receiver>
   <slot>delpath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>505</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addnode()</slot>
  <slot>addpath()</slot>
  <slot>delnode()</slot>
  <slot>delpath()</slot>
 </slots>
</ui>
